### PR TITLE
IGNITE-16225 Read Repair control.sh command should have named params

### PR DESCRIPTION
**Pull request by anton-vinogradov**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java [hunk 1]
```diff
@@ -43,6 +43,24 @@
  *
  */
 public class ConsistencyCommand extends AbstractCommand<Object> {
+    /** Cache. */
+    public static final String C = "-c";
```
Comment (timoninmaxim) [inline modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:47]: As I can see, Ignite doesn't support shortened params for any commands in the control utility. Some operations use the full "--cache" param, but no one uses the "-c" param. Let's do it similarly.

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java [hunk 2]
```diff
@@ -43,6 +43,24 @@
...
+
+    /** Cache. */
+    public static final String CACHE = "--cache";
```
Comment (timoninmaxim) [inline modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:50]: Other commands uses enums for list of params. Let's do it in the same way.

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java [hunk 3]
```diff
@@ -132,10 +150,48 @@
...
+                        default:
+                            throw new IllegalArgumentException("Illegal argument: " + arg);
+                    }
+                }
+                else
+                    break;
+            }
 
-            ReadRepairStrategy strategy = ReadRepairStrategy.fromString(argIter.nextArg("Expected strategy."));
+            assert cacheName != null;
```
Comment (timoninmaxim) [inline modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:192]: Why assert? Let's throw a `IllegalArgumentException` here.

**Review by timoninmaxim — CHANGES_REQUESTED** [review]

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 1 above)
Comment (anton-vinogradov) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:47]: Why not? 
This is useful! Let me be the first! :)

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 3 above)
Comment (anton-vinogradov) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:192]: At least some of the commands are strings:
ClusterStateChangeCommand#FORCE_COMMAND
DefragmentationCommand#NODES_ARG
WalCommands#WAL_PRINT
...

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 2 above)
Comment (anton-vinogradov) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:50]: At least some of the commands are strings:
ClusterStateChangeCommand#FORCE_COMMAND
DefragmentationCommand#NODES_ARG
WalCommands#WAL_PRINT
...

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 1 above)
Comment (timoninmaxim) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:47]: IMHO, it's useful for commands that run frequently in console and frequently used with combination of multiple options, like `tar -cvf`, or `ls -lrt`, or `grep -HIi`.

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 2 above)
Comment (timoninmaxim) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:50]: OK, but it's sad :(

File: modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java (hunk 1 above)
Comment (anton-vinogradov) [reply modules/control-utility/src/main/java/org/apache/ignite/internal/commandline/consistency/ConsistencyCommand.java:47]: It's really harder to ake type at `P` than at `PARTITION`. 
Also, the command is shorter, which is important.

**Review by timoninmaxim — APPROVED** [review]

**Pull request merged**